site/src/pages/TerminalPage/TerminalPage.tsx

```diff
@@ -259,9 +266,11 @@ const TerminalPage: FC = () => {
 				if (disposed) {
 					return; // Unmounted while we waited for the async call.
 				}
-				websocket = new WebSocket(url);
+				websocket = new WebsocketBuilder(url)
+					.withBackoff(new ConstantBackoff(1000))
```

Comment:
I think we should switch this to exponential (ideally with some jitter) soon, if not in this PR.  A fixed reconnect interval is a good way to DDoS yourself :laughing: